airdcpp/airdcpp/Client.cpp

```diff
@@ -435,6 +435,14 @@ void Client::onChatMessage(const ChatMessagePtr& aMessage) noexcept {
 		getHubIdentity().getParams(params, "hub", false);
 		params["hubURL"] = getHubUrl();
 		getMyIdentity().getParams(params, "my", true);
+		auto ou = ClientManager::getInstance()->findOnlineUser(HintedUser(aMessage->getFrom()->getUser(), getHubUrl()), false);
```

Comment:
Can't you just use `aMessage->getFrom()`?

Reply:
Yes, you are right. Done.